changelog.md

```diff
@@ -1,5 +1,13 @@
 # Changelog
 
+## 2.11.0~rc2
+
+* Address translators' feedback on UI strings (#7370)
+* Improve output of Ubuntu Noble pre-migration check script (#7369)
+* Run `securedrop-remove-packages` hourly instead of daily (#7377)
```

Comment:
When we have follow-up PRs in RCs to a new feature I tend to prefer rolling them up into one entry. For example, if we find a bug fixed in PR P2 in a new feature PR P1 in RC1, I'd just put them as "* new feature! (P1, P2)". This is partly for brevity but also to make it clear that the bugfix in P2 is not a production fix. It's good as is, but I'd likely do said rollup for the prod changelog unless there are objections.

Reply:
Yep; I thought we'd do rollup at prod changelog time.